### DISCUSSION TITLE
工具箱新建FTP  链接报错  密码不正确

### DISCUSSION THREAD

**Discussion by Alexzzz2008** [1Panel-dev/1Panel#5391] [Q&A]
### 1Panel 版本

1.10.10

### 问题描述

 安装教程
https://1panel.cn/docs/user_manual/toolbox/ftp/
已安装
﻿
在工具箱中  新建用户名  123  密码123  设置目录
﻿
![Sunny_20240609_163822](https://github.com/1Panel-dev/1Panel/assets/40814658/bce506c9-0821-42a3-8802-93f91f5afc3f)

通过软件连接提示密码不正确

### 重现步骤

同上

**Comment by Alexzzz2008** [comment]
sftpgo 未卸载  是否有冲突?

**Comment by Alexzzz2008** [comment]
[pure-ftpd.zip](https://github.com/user-attachments/files/15751391/pure-ftpd.zip)
配置文件

**Comment by zhengkunwang223** [comment]
我们先在本地复现一下

**Comment by wanghe-fit2cloud** [comment]
自行安装 Pure-FTPd 之后，先通过第三方 FTP 客户端连接，验证是否可用。
